docs/reference/actions.md

```diff
@@ -0,0 +1,12 @@
+# Actions
+
+- _get-initial-password_: Retrieve auto-generated initial password for accessing WordPress admin
+  account. The password is set once during deployment. If the wordpress-k8s charm is configured
+  with `initial_settings` parameters containing `admin_password`, this action has no effect.
+
+- _rotate-wordpress-secrets_: Rotate all WordPress secrets, used for WordPress to encrypt
+  information. The secret fields are used to enhance security by encrypting WordPress data. The
+  following secrets are rotated: auth_key, secure_auth_key, logged_in_key, nonce_key, auth_salt,
+  secure_auth_salt, logged_in_salt, nonce_salt
+  The purpose of this action is to invalidate user sessions under security breach circumstances
```

Comment:
Is the purpose to rotate the secrets or to invalidate? Perhpas this paragraph would be better phrased with something like
```
this action WILL invalidate user sessions forcing the users to log in again. This might be useful under security breach circumstances
```
Also, keep in mind that when documenting, the simpler the grammar, the better

Reply:
Rephrased to simpler phrases:
```
 Invalidate user sessions by rotating the following secrets:
  auth_key, auth_salt, logged_in_key, logged_in_salt, nonce_key, nonce_salt, secure_auth_key,
  secure_auth_salt.
  Users will be forced to log in again. This might be useful under security breach circumstances.
```
Thank you for your suggestion :)